app/controllers/api/projects.php

```diff
@@ -1052,7 +1053,10 @@
             ->setAttribute('url', $url)
             ->setAttribute('security', $security)
             ->setAttribute('httpUser', $httpUser)
-            ->setAttribute('httpPass', $httpPass);
+            ->setAttribute('httpPass', $httpPass)
+            ->setAttribute('enabled', true)
+            ->setAttribute('logs', '')
+            ->setAttribute('attempts', 0);
```

Comment:
Where do you use the param `enable`?

Reply:
<img width="492" alt="Screenshot 2023-12-14 at 7 58 09 AM" src="https://github.com/appwrite/appwrite/assets/20852629/dbeaefc8-eda3-41c0-bc56-d791d453e3e2">
Also looking at other APIs, we use the word `enabled` for the param name. Let's stay consistent with the name 👍

Reply:
It'd be used from the console if the user just wants to `enable` the webhook without updating any other params.